rewriter/Minitest.cc

```diff
@@ -368,13 +474,29 @@ ast::ExpressionPtr prepareTestEachBody(core::MutableContext ctx, core::NameRef e
 }
 
 ast::ExpressionPtr runSingle(core::MutableContext ctx, bool isClass, ast::Send *send, bool insideDescribe) {
+    // Handle include_context first (1+ pos args + optional kwargs/block)
+    if (send->fun == core::Names::includeContext() && send->numPosArgs() >= 1) {
+        auto &arg = send->getPosArg(0);
+        auto argString = to_s(ctx, arg);
+        // Create an include statement that includes the shared_examples companion module
+        // Use proper scoping - if we're inside a describe block, look locally; otherwise use root
+        auto moduleName = ast::MK::UnresolvedConstant(
+            arg.loc(), insideDescribe ? ast::MK::EmptyTree() : ast::MK::Constant(arg.loc(), core::Symbols::root()),
+            ctx.state.enterNameConstant("<shared_examples_module '" + argString + "'>"));
```

Comment:
Do you have a test for this? This seems either redundant or counterintuitive. If we're not in a describe block, using EmptyTree would still allow resolving to root-scoped shared_examples_module constant names.


```suggestion
        auto moduleName = ast::MK::UnresolvedConstant(
            arg.loc(), ast::MK::EmptyTree(),
            ctx.state.enterNameConstant("<shared_examples_module '" + argString + "'>"));
```

Reply:
hmm yeah good point... updating here fdb81cebb